src/frontend/src/pages/PIAForm/index.tsx

```diff
@@ -643,32 +645,34 @@ const PIAFormPage = () => {
   //
   // Form Submission Handler
   //
-  const handleSubmit = (event: any) => {
+  const handleSubmit = (event: React.FormEvent) => {
     event.preventDefault();
-    // if the user is in pia intake tab, then show submit pia intake; else submit full form
+
+    // Determine which modal to show based on `pia` status.
+    const modalMapping = {
```

Comment:
can this be inside the status list logic?

Reply:
Thanks I'll see what I can do